### ISSUE TITLE
SyntaxError: Non-ASCII character '\xe2' in file taxes.py 

### ISSUE THREAD

**Issue by cmcginty** [b-k/py1040#8]
Error when running 'make' command and python version is `2.7.6`:

```
$ make
git clone https://github.com/b-k/1040.js
Cloning into '1040.js'...
remote: Counting objects: 205, done.
remote: Total 205 (delta 0), reused 0 (delta 0), pack-reused 205
Receiving objects: 100% (205/205), 561.19 KiB | 523.00 KiB/s, done.
Resolving deltas: 100% (123/123), done.
Checking connectivity... done.
echo "###AUTOGENERATED via m4. Please edit the sources." > taxforms.py
m4 -P macros/pull_fns.m4 1040.js/forms/f*.m4 1040.js/forms/s*.m4 >> taxforms.py
m4 -P macros/pull_deps.m4 1040.js/forms/f*.m4 1040.js/forms/s*.m4 | sed '$a)' >> taxforms.py
m4 -P macros/pull_nodes.m4 1040.js/forms/f*.m4 1040.js/forms/s*.m4 | sed '$a)' >> taxforms.py
python taxes.py
  File "taxes.py", line 54
SyntaxError: Non-ASCII character '\xe2' in file taxes.py on line 54, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
make: *** [run] Error 1
```

**Comment by scottpaine** [comment]
Issue lies with the funky ASCII characters in taxes.py, these ones:

├, ├───┐, and the │ characters (the last one looks like a pipe, but there's apparently a difference: │ versus |)

I just replaced the characters with regular pipes and minus signs and was able to run the entire thing fine.

**Comment by b-k** [comment]
@cmcginty, thanks for pointing this out. I had been using python3, which is OK with UTF-8, and I'm guessing you are using python2.7, which requires an encoding string.

Thanks, @scottpaine for localizing this in `print_the_tree`---which doesn't even run: it was an early draft of a visualization that I commented out a long time ago. To address this issue, I removed it entirely in version ad7a0c1, so it should no longer be an issue.
